### DISCUSSION TITLE
Raspberry pi installation, enable web ui

### DISCUSSION THREAD

**Discussion by dstrimble** [cagnulein/qdomyos-zwift#1730] [Q&A] [answered]
Is there a way to hit the config UI from a putty session or webpage after installing on raspberry pi?

**Comment by dstrimble** [comment]
I tried these instructions and it didn't work.

https://github.com/cagnulein/qdomyos-zwift/discussions/1685#discussioncomment-7167894

**Comment by cagnulein** [comment]
@dstrimble which settings do you want to change? you can edit the .conf file directly if you want

**Reply by dstrimble** [reply]
I was hoping to get my HRM to connect through QZ so that I don't have to connect it seperately in zwift. Without the UI I can't even tell if it's connected already.

**Reply by cagnulein** [reply]
@dstrimble raspberry doesn't have ANT+ and it doesn't handle the companion. The only way is with QZ on iOS or Android. Share a debug log of your current setup on ios or android with the companion and I will check it https://github.com/cagnulein/qdomyos-zwift/wiki/How-do-i-get-the-debug-log-in-case-something-doesn't-work%3F

**Reply by dstrimble** [reply]
HRM has bluetooth so I was hoping it would work. Thanks.

**Reply by cagnulein** [reply, marked as answer]
@dstrimble in the config file change the heart_rate_belt_name setting with the exact bluetooth name of your hrm device (case sensitive)

**Reply by dstrimble** [reply]
This worked for the Bluetooth HRM, thanks.
